source/style-guide.txt

```diff
@@ -31,14 +31,6 @@ MongoDB external and internal customers.
    Provides instructions for when to use and how to create screenshots
    and diagrams to enhance your content.
 
-:doc:`/style-guide/how-to-article-guidelines`
```

Comment:
**suggestion:** Maybe don't delete this section, but:
1. Change it to point to the Guides page or
2. Change it to point to the Tasks page instead?

The Guides page probably needs to move, too. This may need to be revisited.